src/assets/server/rest/v1/schemas/chargingstation/chargingprofile-create.json

```diff
@@ -1,5 +1,115 @@
 {
   "title": "Create a Charging Profile",
   "type": "object",
-  "$ref": "chargingstation.json#/definitions/chargingProfile"
```

Comment:
Is it possible to reuse the profile structure from the definition somehow?
The only thing, which differs is that the id is mandatory right?

I also added some enums of the values which are possible for the different cp properties